rust/src/io/object_store.rs

```diff
@@ -139,6 +146,12 @@ impl ObjectStore {
                 base_path: Path::from(url.path()),
                 block_size: 64 * 1024,
             }),
+            "az" => Ok(Self {
```

Comment:
Should we support all URL schemes described here

https://docs.rs/object_store/latest/object_store/azure/struct.MicrosoftAzureBuilder.html#method.with_url

Reply:
IMO
1/ we should avoid http and https (generic)
2/ abfs ones are mainly for Hadoop
3/ adl is for Gen 1
4/ azure is not on the spec

This why, I restricted it to az.

Reply:
I see. Thanks for the explanation.